compiler/codegen/CodeGenRA.cpp

```diff
@@ -77,6 +77,9 @@
 #include "ras/Debug.hpp"
 
 #define OPT_DETAILS "O^O CODE GENERATION: "
+#define OPT_DETAILS_SPILL_TEMPS "O^O SPILL TEMPS: "
+#define OPT_DETAILS_HALF_SLOT_SPILLS "O^O HALF-SLOT SPILLS: "
+#define OPT_DETAILS_GLOBAL_REGISTER_ALLOCATION "O^O GLOBAL REGISTER ALLOCATION: "
```

Comment:
Doesn't look like we use `OPT_DETAILS` anywhere so maybe just update to say `CODE GENERATION REGISTER ALLOCATION: `.